### PR TITLE
Make more readable error strings.

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

**Comment by humblec** [comment]
@wongma7 please review. I am yet to check the travis failure is spurious.

File: gluster/block/cmd/glusterblock-provisioner/glusterblock-provisioner.go [hunk 1]
```diff
@@ -363,27 +367,31 @@ func (p *glusterBlockProvisioner) Delete(volume *v1.PersistentVolume) error {
...
 	case "heketi":
-		glog.V(1).Infof("heketik: Deleteing Volume %v", delBlockVolName)
+		glog.V(1).Infof("glusterblock: opmode[heketi]: deleting Volume %v", delBlockVolName)
 	default:
-		glog.Errorf("Unknown OpMode, failed to delete volume %v", delBlockVolName)
+		glog.Errorf("glusterblock: Unknown OpMode, failed to delete volume %v", delBlockVolName)
 
 	}
 
+	deleteSecErr := p.client.Core().Secrets(p.volConfig.BlockSecretNs).Delete(p.volConfig.BlockSecret, nil)
```
Comment (wongma7) [inline gluster/block/cmd/glusterblock-provisioner/glusterblock-provisioner.go]: Won't other PVs rely on the secret? I thought it was one secret per Storage Class

File: gluster/block/cmd/glusterblock-provisioner/glusterblock-provisioner.go [hunk 2]
```diff
@@ -245,7 +247,9 @@ func (p *glusterBlockProvisioner) createSecretRef(nameSpace string, secretName s
 		Type: chapType,
 	}
 
+	p.volConfig.BlockSecret = secretName
 	secretRef := &v1.LocalObjectReference{}
+	p.volConfig.BlockSecretNs = nameSpace
 	if secret != nil {
 		_, err = p.client.Core().Secrets(nameSpace).Create(secret)
```
Comment (wongma7) [inline gluster/block/cmd/glusterblock-provisioner/glusterblock-provisioner.go]: What if the secret already exists? Isn't there a very high chance of that if Provisioning multiple PVs? I was under the impression they all use the same secret (storage class immutable)

Comment (wongma7) [inline gluster/block/cmd/glusterblock-provisioner/glusterblock-provisioner.go]: Correction: I mean if provisioning pvs for multiple pvcs *in the same namespace*. Same idea though.

File: gluster/block/cmd/glusterblock-provisioner/glusterblock-provisioner.go (hunk 1 above)
Comment (wongma7) [inline gluster/block/cmd/glusterblock-provisioner/glusterblock-provisioner.go]: one secret per Storage Class per namespace *

**Comment by wongma7** [comment]
the travis failure is nfs's fault, my 'everything-else' suite passed.

File: gluster/block/cmd/glusterblock-provisioner/glusterblock-provisioner.go (hunk 1 above)
Comment (humblec) [inline gluster/block/cmd/glusterblock-provisioner/glusterblock-provisioner.go]: @wongma7 its per PV and it holds the user credentials of that block volume. At time of PV deletion it should go away.

File: gluster/block/cmd/glusterblock-provisioner/glusterblock-provisioner.go (hunk 2 above)
Comment (humblec) [inline gluster/block/cmd/glusterblock-provisioner/glusterblock-provisioner.go]: @wongma7  Actually secrets are created with a unique identifier, that said, the `user` get a unique ID from gluster block.  so I believe we are safe here.

**Comment by wongma7** [comment]
Secret name is not unique though if multiple pvs can have the same user. Is
that possible?

**Comment by humblec** [comment]
@wongma7 the user field is filled with a value called 'GBID' which is unique identifier generated by the gluster block internally.

**Comment by wongma7** [comment]
Ok, I must have confused cfg.user (class param) with vol.user (from response).

Lgtm

**Pull request merged**